INSTALL.md

```diff
@@ -16,10 +16,11 @@ Prerequisites:
 * pkg-config (Debian package: pkg-config)
 * libical >= 1.0.0 (Debian package: libical-dev)
 * xsltproc (Debian package: xsltproc)
+* gpgme
 
 Install these prerequisites on Debian GNU/Linux 'Buster' 10:
 
-    apt-get install gcc cmake libglib2.0-dev libgnutls28-dev libpq-dev postgresql-server-dev-11 pkg-config libical-dev xsltproc
+    apt-get install gcc cmake libglib2.0-dev libgnutls30-dev libpq-dev postgresql-server-dev-11 pkg-config libical-dev xsltproc libgpgme-dev
```

Comment:
The dev package is named libgnutls28-dev. The binary package for the .so file is named libgnutls30. Very confusing.
```suggestion
    apt-get install gcc cmake libglib2.0-dev libgnutls28-dev libpq-dev postgresql-server-dev-11 pkg-config libical-dev xsltproc libgpgme-dev
```